package.json

```diff
@@ -7,28 +7,30 @@
   "author": "Simon Højberg <r.hackr@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/public/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
-    "test": "jest",
+    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
+    "test": "rm -rf dist; rm -rf build; jest",
     "tsc": "tsc",
     "prepublishOnly": "rm -rf dist; tsc --outDir dist/ --module commonjs"
   },
   "peerDependencies": {
-    "react": ">=15.0.0"
+    "react": ">=16.0.0"
```

Comment:
@newyankeecodeshop What version of React is the Point of Sale on?

Reply:
@hojberg We are on React 16.8.6, which is the version that React Native 0.60 embeds.

Reply:
Perfect. I'm prolly gonna do a major release today for that requirement, but I think it should be safe for Upserve to upgrade.